### PR TITLE
Build/node 18

### PR DESCRIPTION
**Pull request by frenautvh**
| Question         | Answer
| ---------------- | ---
| Branch?          | master
| Bug fix?         | no
| New feature?     | no
| Breaking change? | yes
| Tickets          | DTCORE-78
| License          | BSD 3-Clause

## Description

Bump monorepo to node 18 & webpack 5

### CODE REVIEW

**Review by anooparveti — APPROVED** [review]

**Review by antleblanc — CHANGES_REQUESTED** [review]
Could you please resolve existing conflicts? Thank you!

**Comment by frenautvh** [comment]
rebased on master de02b1600dafb80ad48395ccc4576049991fb46f

**Comment by frenautvh** [comment]
rebased on master a84bf63d00bd3ce3f477635129436c45bf4d0c81

**Comment by cbourgois** [comment]
Rebased on `master`

**Review by antleblanc — APPROVED** [review]

**Review by antleblanc — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/creating/index.scss [hunk 1]
```diff
@@ -123,7 +123,7 @@ pci-project-creating {
 
     50% {
       stroke-dasharray: 120, 300;
-      stroke-dashoffset: -$d / 3;
+      stroke-dashoffset: calc(-#{$d} / 3);
```
Comment (qpavy) [inline packages/manager/modules/pci/src/projects/creating/index.scss:126]:
```suggestion
      stroke-dashoffset: calc(-$d / 3);
```
Why need string encapsulation here as $d is a number ?

File: packages/manager/modules/pci/src/projects/project/creating/creating.scss [hunk 2]
```diff
@@ -108,7 +108,7 @@ pci-project-creating {
 
     50% {
       stroke-dasharray: 120, 300;
-      stroke-dashoffset: -$d / 3;
+      stroke-dashoffset: calc(-#{$d} / 3);
```
Comment (qpavy) [inline packages/manager/modules/pci/src/projects/project/creating/creating.scss:111]: same as https://github.com/ovh/manager/pull/8679/files#r1289693974

**Review by antleblanc — DISMISSED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Review by antleblanc — DISMISSED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Review by BrunoMarquesOVH — APPROVED** [review]

**Pull request merged**